lib/fpjs_pro_plugin.dart

```diff
@@ -28,6 +28,8 @@ class FpjsProPlugin {
       String? scriptUrlPattern,
       List<String>? scriptUrlPatternFallbacks,
       Region? region,
+      bool? allowUseOfLocationData,
+      int? locationTimeoutMillis,
```

Comment:
If this only applies to Android, maybe we can make it obvious from the parameter name? 

```suggestion
      int? locationTimeoutMillisAndroid,
```